src/main.rs

```diff
@@ -85,15 +112,28 @@ fn run_with_file_map(conf: Config, log: Logger) -> Result<(), Error> {
             file_map.insert(fname, file);
         }
     }
+    if conf.multithreaded {
+        run_multithreaded(conf, log, move |msg| {
+            let shark = msg.shark.replace(domain_prefix.clone().as_str(), "");
```

Comment:
Why not just `let shark = msg.shark.replace(&domain_prefix, "");` ?

Reply:
Of course.